Core/src/org/sleuthkit/autopsy/textextractors/TikaTextExtractor.java

```diff
@@ -171,9 +169,29 @@
      *
      * @return Flag indicating if OCR is set to be used.
      */
-    private boolean ocrEnabled() {
-        return TESSERACT_PATH != null && tesseractOCREnabled
-                && PlatformUtil.isWindowsOS() == true && PlatformUtil.is64BitOS();
+    /**
+     * Method to indicate if OCR should be performed on this image file. Checks
+     * to see if the limited OCR setting is enabled. If it is it will also check
+     * that one of the limiting factors is true.
+     *
+     * @param file    The AbstractFile which OCR might be performed on.
+     * @param boolean The configuration setting which indicates if limited OCR
+     *                is enabled in Keyword Search.
+     *
+     * @return True if limited OCR is not enabled or the image is greater than
+     *         100KB in size or the image is a derived file.
+     */
```

Comment:
I don't think this comment is accurate at this point. One of the goals that emerged for for this story was to insulate the text extractors from the "limited OCR" workaround.